build/Build.cs

```diff
@@ -97,7 +97,7 @@ class Build : NukeBuild
                 .DisableIncludeSymbols()
                 .SetVerbosity(DotNetVerbosity.Normal)
                 .SetProperty("NuspecFile", "../../build/Octopus.Server.Extensibility.Authentication.Ldap.nuspec")
-                .SetProperty("NuspecProperties", $"Version={OctoVersionInfo.NuGetVersion}"));
+                .SetProperty("NuspecProperties", $"Version={OctoVersionInfo.FullSemVer}"));
```

Comment:
Using nugetversion was using te trnucated version on this long branch name, causing the nupkg file to not be found when pushing to octopus. First changed the build to use the nugerversion but this caused issues because server has a nuspec file and client doesnt meaning only one of the 2 had the shorter nuget version being used.  Actually it seems these days nuget doesnt mind if we use the longer version tags, so this change uses FullSemVer everywhere and keeps server and client naming etc inline with eachother.